### PR TITLE
Support configuring broker version in Kafka bindings component

### PR DESCRIPTION
**Pull request by light-pan**
# Description

 Kafka bindings to support the Version metadat

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1277

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[2139]_(https://github.com/dapr/docs/issues/2139

### CODE REVIEW

File: bindings/kafka/kafka.go [hunk 1]
```diff
@@ -198,6 +201,17 @@ func (k *Kafka) getKafkaMetadata(metadata bindings.Metadata) (*kafkaMetadata, er
 		meta.MaxMessageBytes = maxBytes
 	}
 
+	if val, ok := metadata.Properties["version"]; ok && val != "" {
+		version, err := sarama.ParseKafkaVersion(val)
+
+		if err != nil {
+			return nil, errors.New("kafka error: invalid kafka version")
```
Comment (Taction) [inline bindings/kafka/kafka.go:208]: return nil, fmt.Errorf("kafka error: invalid 'version' attribute: %w", err)

**Comment by Taction** [comment]
@light-pan Thanks for your contribution. Please fix lint issues. Search for "Check failure" here: https://github.com/dapr/components-contrib/pull/1459/files. it'll show you exactly which files need what formatter.

File: bindings/kafka/kafka.go (hunk 1 above)
Comment (light-pan) [reply bindings/kafka/kafka.go]: Done

**Review by Taction — APPROVED** [review]
LGTM

**Review by daixiang0 — APPROVED** [review]

**Pull request merged**